File: 백준/Silver/2563. 색종이/색종이.py
import sys

area = [0 for i in range(10000)]

n = int(sys.stdin.readline())

for i in range(n):
    x, y = map(int, sys.stdin.readline().split())
    z = 100 * y + x
    for j in range(z, z + 1000, 100):
        for k in range(10):
            area[j + k] = 1

print(area.count(1))

File: 백준/Silver/2563. 색종이/색종이.js
const rl = require('readline').createInterface({
    input: process.stdin,
    output: process.stdout,
});

let paper = Array.from({ length: 100 }, () => Array(100).fill(0));
let coloredArea = 0;

rl.on('line', (line) => {
    const [x, y] = line.split(' ').map(Number);
    for (let i = x; i < x + 10; i++) {
        for (let j = y; j < y + 10; j++) {
            if (paper[i][j] === 0) {
                paper[i][j] = 1;
                coloredArea++;
            }
        }
    }
}).on('close', () => {
    console.log(coloredArea);
    rl.close();
});
